core/templates/dev/head/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.directive.html

```diff
@@ -73,6 +73,25 @@ <h3>Reload a single collection</h3>
   </div>
 </md-card>
 
+<md-card class="oppia-page-card oppia-long-text" style="max-width: 700px">
+  <div class="container" style="margin-left: 0;">
+    <h3>Load dummy new structures data (only admins)</h3>
+    <div class="row">
+      <span class="col-lg-4 col-md-4 col-sm-4">
+        Loads a topic, a story and three skills (two in a subtopic) attached to the topic
```

Comment:
Note that if we decide to auto-load two topics, this documentation needs to be updated.

Reply:
Done